component/file_cache/file_cache.go

```diff
@@ -750,14 +882,7 @@ func (fc *FileCache) downloadFile(handle *handlemap.Handle) error {
 			fileSize = int64(attr.Size)
 		}
 
-		if fileExists {
```

Comment:
Could you explain this change? I believe this section is intended to get the latest copy from the cloud when download file is called so the local copy is first cleared.

Reply:
Yes that is correct, but I remember we were running into an issue with this deleting files when the cloud was down, so the local copy of the file gets deleted but there's no cloud to copy from. When retrieving the cloud copy of the file it will just overwrite the local copy so we don't need to delete. Instead, just check for file existence and make the file if it doesn't exist already.